src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -79,6 +74,11 @@ interface Props {
   intl: InjectedIntl;
 }
 
+// Providing a buffer between the limit and the cut off index
+// protects from truncating just the last couple (6) characters
+const TRUNCAT_LIMIT: number = 64;
```

Comment:
Is this missing an "E" or am I reading this wrong?

Reply:
no, you're right....